src/lib/viewers/BaseViewer.js

```diff
@@ -361,6 +354,20 @@ class BaseViewer extends EventEmitter {
     }
 
     /**
+     * Applies appropriate styles and resizes the document depending on fullscreen state
+     * @protected
```

Comment:
Nit - space after description before @. We also stopped marking things as protected